src/main/java/serverutils/data/ClaimedChunks.java

```diff
@@ -150,6 +153,14 @@ public void removeChunk(ChunkDimPos pos) {
             prevChunk.setInvalid();
             markDirty();
         }
+        if (Loader.isModLoaded(OtherMods.VP)) {
```

Comment:
nit: Should cache this in a bool during one of the init phases instead of calling isModLoaded every time

Reply:
Good call